doc/iro.txt

```diff
@@ -0,0 +1,43 @@
+*iro.txt*		a next generation syntax highlighter
+
+Version: 0.1.0-dev
+Author:  Masataka Pocke Kuwabara
+Support: Vim 8.0.0000 or newer with if_ruby
+License: MIT License (See LICENSE)
```

Comment:
Iro.vim's license is not MIT, it is Apache v2 https://github.com/pocke/iro.vim/blob/master/LICENSE

Reply:
Oooooops! Sorry for that. I fixed this by `git amend` in my fork side, to make sure upstream iro.vim repository keeps same license for every commit. It's now `Apache v2`.